gnmvidispine/vs_item.py

```diff
@@ -903,6 +903,21 @@ def parent_collections(self, shouldPopulate=False):
                 cref.name = uri_entry.text
             yield cref
 
+    def to_cache(self):
+        """
+        Returns a dictionary of the data in the object
+        :return: dictionary of data in the object
+        """
+        return self.contentDict
```

Comment:
likewise i would suggest returning a deep-copy, otherwise if some other process modifies this dictionary it would change the internal state of the object too

Reply:
Will do.